webview/src/plots/components/ribbon/RibbonBlockIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from './styles.module.scss'
+import { Error, Info } from '../../../shared/components/icons'
+
+export const RibbonBlockIcon: React.FC<{
+  hasError: boolean
+}> = ({ hasError }) =>
+  hasError ? (
+    <Error
+      width={14}
+      height={14}
+      className={cx(styles.blockIcon, styles.error)}
+      aria-label="Error Icon"
```

Comment:
Technically, you can label svgs but we don't really do in the extension. I added labels for testing but there might be a better way of handling this.